tests/Unit/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProviderTest.php

```diff
@@ -607,6 +610,64 @@ private function getDatasetsForCategories(): array
         return $datasets;
     }
 
+    /**
+     * @return array
+     */
+    private function getDatasetsForPackedProducts(): array
+    {
+        $datasets = [];
+
+        $expectedOutputData = $this->getDefaultOutputData();
+        $productData = [
+            'packed_products' => [
+                0 => [
+                    'product_id' => 15,
+                    'productName' => 'wicked snowboard',
+                    'quantity' => 3,
+                    'reference' => 'demo_15',
+                    'combination_id' => null,
```

Comment:
combination ID should never be null, but 0 is accepted

Reply:
It's always me me sad to put a 0 where a null have to be .... It's legacy ... but it's sad.